Integrations/EWSv2/EWSv2.py

```diff
@@ -1555,6 +1560,30 @@ def get_autodiscovery_config():
     }
 
 
+def mark_item_as_read(item_ids, operation='read', target_mailbox=None):
+    marked_items = []
+    account = get_account(target_mailbox or ACCOUNT_EMAIL)
+    if type(item_ids) != list:
```

Comment:
use `argToList` instead
```suggestion
    item_ids = argToList(item_ids)
```